src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -594,7 +596,7 @@ partial class BoundIncrementOperator : IIncrementExpression
 
     partial class BoundBadExpression
     {
-        protected override OperationKind ExpressionKind => OperationKind.None;
+        protected override OperationKind ExpressionKind => OperationKind.BadExpression;
```

Comment:
"Bad" seems like a really "bad" name.  Why not "Invalid" or something lke that.  I